.bazelci/config.yaml

```diff
@@ -73,4 +106,21 @@ tasks:
     platform: ubuntu2004
     working_directory: docs
     build_targets:
-    - //...
\ No newline at end of file
+    - //...
+  min_supported_version:
+    name: "Minimum Supported Version"
+    bazel: "3.4.0"
+    platform: ubuntu1604
```

Comment:
I assume this is 1604 because this is the only place where 3.4.0 is available in CI?

Reply:
I don't think so. I didn't want to drop support for 1604 here so I use it for the min version tests. If we decide we don't aren't going to test on that environment, I can easily change it but wanted that to be a well communicated decision.